src/models/neural_networks/ealstm.py

```diff
@@ -288,11 +289,14 @@ def forward(
 
         x = self.rnn_dropout(hidden_state[:, -1, :])
 
+        if return_embedding:
```

Comment:
is this for interpreting the static embedding?

Reply:
No - the loss in tile2vec compares the embedding, not the final value. This is to return that embedding for the loss, before it gets put through the final linear layer

Reply:
yes makes sense! Could this be used for interpreting the embedding layer too though?

Reply:
yea, 100%. Although here the "embedding" is the final output of the model before the linear regression layer